lib/docs/filters/tailwindcss/entries.rb

```diff
@@ -0,0 +1,18 @@
+module Docs
+  class Tailwindcss
+    class EntriesFilter < Docs::EntriesFilter
+      def get_type
+        # /customizing-colors rediects to /colors, hence making css
+        # selector below not to match the href
+        if result[:path] == 'customizing-colors'
+          selector = "#sidebar a[href='#{result[:path]}']"
+        else
+          selector = "#sidebar a[href='#{result[:path]}']"
```

Comment:
These lines look like they do the same thing.

Reply:
Oops! Forgot that. However, this conditional selector is no longer necessary because the redirect issue that necessitated this has been taken care of upstream (original TailwindCss documentation)